### PR TITLE
docs: Add a note on goal of writing tests

### PR DESCRIPTION
**Pull request by srisankethu**
Fixes https://github.com/coala-analyzer/coala/issues/1999

### CODE REVIEW

File: docs/Getting_Involved/Writing_Tests.rst [hunk 1]
```diff
@@ -9,7 +9,12 @@ characteristic to be undiscoverable (or only discoverable after dozens
 of hours of searching). Try to test as much as possible! The more tests
 you write the more you can be sure you did everything correct.
 Especially if someone else modifies your component, he can be sure with
-your tests that he doesn't introduce a bug.
+your tests that he doesn't introduce a bug. Keep these points in your mind
+when your writing your writing a test:
```
Comment (Makman2) [inline docs/Getting_Involved/Writing_Tests.rst]: doubled "your writing" :)
also shouldn't it be `you're writing`?

File: docs/Getting_Involved/Writing_Tests.rst [hunk 2]
```diff
@@ -9,7 +9,12 @@ characteristic to be undiscoverable (or only discoverable after dozens
...
+
+- 100% test-coverage
+- zero redundancy
```
Comment (Makman2) [inline docs/Getting_Involved/Writing_Tests.rst]: I think you can describe these a bit^^

Comment (Makman2) [inline docs/Getting_Involved/Writing_Tests.rst]: Especially a reason would be good **why** we want this :)

**Comment by Makman2** [comment]
unack b26c4a5

**Comment by srisankethu** [comment]
@Makman2  Ya sure ! will do it right now

**Comment by srisankethu** [comment]
@Makman2  Updated !

**Comment by tushar-rishav** [comment]
LGTM :+1:

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2001/commits/08466ba2122550eea406eed6f31162170668e60f then

**Comment by sils1297** [comment]
@rultor merge (@tushar-rishav feel free to ack and merge simple things on your own btw.)

**Pull request merged**